docs/source/search-api-user-guide.rst

```diff
@@ -0,0 +1,21 @@
+PDS Search API User Guide
+=========================
+
+The Planetary Data System is a federated system of nodes that archive planetary science data. The PDS Application Programming
+Interface (API) is an attempt at provide a consistent interface for sharing archival data and services across PDS, among
```

Comment:
Maybe just say:

> The PDS Application Programming Interface (API) provides a consistent way discover and share archival data across PDS…

The wording of "attempt to provide" makes it sound weak 🤔

Reply:
Agreed and reworded.